src/test/java/step2/CalculatorTest.java

```diff
@@ -1,75 +1,17 @@
 package step2;
 
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 public class CalculatorTest {
 
-    @DisplayName("덧셈 : 10과 15의 덧셈 결과는 25를 반환한다")
-    @Test
-    public void add_calculateTest() {
-        //given
-        int number1 = 10;
-        int number2 = 15;
-        String operator = "+";
-        int expectedValue = 25;
-
-        //when
-        Calculator calculator = new Calculator(number1);
-
-        //then
-        assertThat(calculator.calculate(operator, number2)).isEqualTo(expectedValue);
-    }
-
-    @DisplayName("뺄셈 : 31과 15의 뺄셈 결과는 16을 반환한다")
-    @Test
-    public void subtract_calculateTest() {
-        //given
-        int number1 = 31;
-        int number2 = 15;
-        String operator = "-";
-        int expectedValue = 16;
-
-        //when
-        Calculator calculator = new Calculator(number1);
-
-        //then
-        assertThat(calculator.calculate(operator, number2)).isEqualTo(expectedValue);
-    }
-
-    @DisplayName("곱셈 : 11과 16의 곱셈 결과는 176을 반환한다")
-    @Test
-    public void multiply_calculateTest() {
-        //given
-        int number1 = 11;
-        int number2 = 16;
-        String operator = "*";
-        int expectedValue = 176;
-
-        //when
-        Calculator calculator = new Calculator(number1);
-
-        //then
-        assertThat(calculator.calculate(operator, number2)).isEqualTo(expectedValue);
-    }
-
-    @DisplayName("나눗셈 : 8와 2의 나눗셈 결과는 4를 반환한다")
-    @Test
-    public void divide_calculateTest() {
-        //given
-        int number1 = 8;
-        int number2 = 2;
-        String operator = "/";
-        int expectedValue = 4;
-
-        //when
+    @ParameterizedTest(name = "{0} {1} {2} 의 결과값은 {3}을 반환한다.")
+    @CsvSource(value = {"10:+:15:25", "31:-:15:16", "11:*:16:176", "9:/:2:4", "8:/:2:4"}, delimiter = ':')
```

Comment:
```suggestion
    @CsvSource({"10,+,15,25", "31,-,15,16", "11,*,16,176", "9,/,2,4", "8,/,2,4"})
```
혹시 `:` 으로 구분자를 만든 이유가 있을까요? 어떤 형태가 더 가독성 좋은지 이야기 나눠보면 좋겠습니다. 😀

Reply:
`:`를 구분자로 사용한건 딱히 이유는 없습니다. 
아마 제 개인적인 습관일지도 모르는데 `,`보다는 `:`가 읽기 쉽다고 느끼고 있는 것 같습니다.
`,`는 배열값, 파라메터 값의 구분 예약어로 사용되고 있기 때문에 중복되어 있는 느낌이라 그런거 같네요.

이 문제에 대해서는 한번도 생각해 보진 않았는데 `,`를 사용하는게 더 일반적일까요?

인사이트를 주셔서 감사합니다.😀